File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/opengl/renderer/GL30Renderer.java
package br.com.insanitech.spritekit.opengl.renderer;

import javax.microedition.khronos.egl.EGLConfig;
import javax.microedition.khronos.opengles.GL10;

/**
 * Created by anderson on 6/29/15.
 */
public class GL30Renderer extends GL20Renderer {
    @Override
    public void onSurfaceCreated(GL10 gl, EGLConfig config) {
        super.onSurfaceCreated(gl, config);
    }

    @Override
    public void onSurfaceChanged(GL10 gl, int width, int height) {
        super.onSurfaceChanged(gl, width, height);
    }

    @Override
    public void onDrawFrame(GL10 gl) {
        super.onDrawFrame(gl);
    }
}

File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/opengl/renderer/GL30Renderer.kt
package br.com.insanitech.spritekit.opengl.renderer

import javax.microedition.khronos.egl.EGLConfig
import javax.microedition.khronos.opengles.GL10

/**
 * Created by anderson on 6/29/15.
 */
internal class GL30Renderer(drawer: GLDrawer) : GL20Renderer(drawer) {
    override fun onSurfaceCreated(gl: GL10, config: EGLConfig) {
        super.onSurfaceCreated(gl, config)
        TODO("implement GL ES 3.0 support")
    }

    override fun onSurfaceChanged(gl: GL10, width: Int, height: Int) {
        super.onSurfaceChanged(gl, width, height)
        TODO("implement GL ES 3.0 support")
    }

    override fun onDrawFrame(gl: GL10) {
        super.onDrawFrame(gl)
        TODO("implement GL ES 3.0 support")
    }
}
